tests/ha/test_pod_failure.py

```diff
@@ -3484,3 +3484,75 @@ def test_chunk_upload_during_pod_down(self):
         LOGGER.info("Step 10: Successfully completed IOs.")
 
         LOGGER.info("ENDED: Test chunk upload during pod down")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.nightly_regression
+    @pytest.mark.tags("TEST-39976")
+    @CTFailOn(error_handler)
+    def test_object_crud_pod_failure(self):
+        """
+        Verify object CRUDs before and after pod failure; pod shutdown by making replicas=0
+        """
+        LOGGER.info("STARTED: Verify object CRUDs before and after pod failure; pod shutdown "
+                    "by making replicas=0")
+
+        LOGGER.info("Step 1: Perform WRITEs-READs-Verify with variable object sizes. 0B + ("
+                    "1KB - 512MB)")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-39976'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix, skipcleanup=True)
```

Comment:
For the s3bench log_prefix, keep it TEST-39976, since while collecting s3bench logs from pytest, we are looking for that.
Check `upload_supporting_logs` function for more info.

Reply:
> For the s3bench log_prefix, keep it TEST-39976, since while collecting s3bench logs from pytest, we are looking for that. Check `upload_supporting_logs` function for more info.

@ketanarlulkar , as per [https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html](url), we should not uppercase in bucket name creation. As we are using self.test_prefix, in generating bucket, we have followed this format.

Reply:
Understand that. But log_file_prefix in the s3bench function we should pass TEST-39976. If we don't, then IMU pytest will not collect s3bench log file, after the execution.

Reply:
As per discussion, done the changes.